### ISSUE TITLE
2021 - Aktualisierung Modul B2

### ISSUE THREAD

**Issue by karbecke** [dzhw/SiD#311]
Liebes ZOFAR-Team,

das Modul B2 wurde aktualisiert:

- [ ] Bitte arbeitet die Änderungen von ab63429 in die QML ein.

Ebenso sind uns noch einige notwendige Anpassungen der QML (auf Basis der bereitgestellten presenter-Version) aufgefallen, die im GitHub-Code schon richtig stehen, aber noch in die QML eingearbeitet werden müssten:

- [ ] B2_3a: Bitte mv umsetzen
- [ ] B2_8a2: Seite "B2_8a" wird neu zu Seite "B2_8a2"
- [ ] B2_9b: Items und Skala in QML bitte gemäß GitHub-Code anpassen. Skala und Items bitte tauschen
- [ ] B2_11: Seite wird neu angelegt

Vielen Dank!

**Comment by SilinaKowalewski** [comment]
@karbecke:

**B2_b2**: Uneinheitliche Kommasetzung von st5 und st6 (vor dem "und")

> (st5: !!Selbstständiger im Handel, Gewerbe, Handwerk, Industrie, Dienstleistung und zwar …!! (ao17, ao18, ao19) - st6: !!Selbständiger Landwirt bzw. Genossenschaftsbauer, und zwar …!! (ao20, ao21, ao22))

**Comment by karbecke** [comment]
@SilinaSchirmer : Ja, ist wohl durchgerutscht. Bitte das Komma noch ersetzen. Wie wollen wir mit diesen Änderungen im weiteren Verlauf verfahren?

**Comment by SilinaSchirmer** [comment]
Alle Änderungen wurden umgesetzt und sind ausgerollt. Ihr könnt also nun mit dem Testen starten.

Wie wir mit weiteren Änderungen vorgehen sollten wir vielleicht am Donnerstag nochmal besprechen.

**Comment by lenamzimmer** [comment]
Liebe Silina,

uns ist noch das Fehlen eines Kommas bei B2_2b aufgefallen:

- [ ] e5aac79ff91fe11270ab7438f74269de8383fe8b

Danke!

**Comment by mcehrhardt** [comment]
Liebe @SilinaSchirmer,

hier fehlt tatsächlich nur noch das Komma aus dem letzten commit (e5aac79), wenn dies umgesetzt ist, kann das Issue geschlossen werden. Könntest du das bitte machen?
LG
Marie

**Comment by SilinaSchirmer** [comment]
Ohje, die Info von @lenamzimmer hatte ich ganz übersehen. Sorry!

Ist aber nun umgesetzt und ab dem nächsten Rollout verfügbar (#391)
